sdk/opendp/smartnoise/sql/private_reader.py

```diff
@@ -10,7 +10,7 @@
 from opendp.smartnoise._ast.expressions import sql as ast
 from opendp.smartnoise.reader import Reader
 
-from opendp.smartnoise.mechanisms.gaussian import Gaussian
+from opendp.smartnoise._mechanisms.gaussian import Gaussian
```

Comment:
it might be better to move _mechanisms here

opendp/smartnoise/sql/_mechanisms.py

And to use the relative import ._mechanisms

No other part of the code base uses it and we don't expect further use right?

Reply:
That's correct; I've moved it